### ISSUE TITLE
Background information

### ISSUE THREAD

**Issue by danielsz** [jamesmacaulay/zelkova#5]
For those that know Elm, presenting the library as Elm-derived FRP probably makes sense.
For people new to FRP like myself, not so much. Not your fault, and I'm doing my homework, but it takes time and the term FRP appears to be overloaded. 

Any effort to make the concepts clear and distinct would be highly appreciated. 
Also, I think it is important to point at differences with existing libraries which play in the same field, such as reagi. 

Thank you so much.

**Comment by jamesmacaulay** [comment]
You're absolutely right, these are good suggestions. At the very least the README should have some actual explanations of things. I just added a link to my talk from Clojure/West, which doesn't go deep into the concepts but hopefully is helpful in other ways.

I'll close this issue when I consider the README "satisfactory", which I currently do not :P

**Comment by danielsz** [comment]
I saw and enjoyed the Clojure/West presentation. Thank you, @jamesmacaulay. 

I especially liked the hybrid Om/Zelkova app, because it fits snugly in oft-encountered use cases.
Because all of this is new to many people, there should be a section in the README called `learning`, and it should have pointers to resources where one can build an understating of the concepts.
Maybe it's possible to reuse Elm learning resources. For example, http://elm-lang.org/Learn.elm
Maybe with some charts documenting the equivalences (e.g `foldp` and `reductions`).

**Comment by chrismurrph** [comment]
Agreed it is hard to start especially for those who are new to Clojure/script too. Where is the best place to ask questions? SO, your email address? I am trying to create a signal from scratch in zelkova_quil_flyer, so:
```clojure
    (defn my-stream
      []
      (z/constant [10,10]))
```
, and just put that signal in in-place-of of `mouse/position` in `app-signal`. What am I doing wrong?
